TWLight/applications/templates/applications/application_evaluation.html

```diff
@@ -221,21 +221,41 @@ <h2>{% trans 'Application' %}</h2>
   {% comment %}Translators: This is the title of the section of an application page which contains the Recent Applications made by the applicant{% endcomment %}
   <h2>{% trans 'Recent Applications' %}</h2>
```

Comment:
Should we change this to `Recent Applications in the Last 90 Days`, or should we mention that the applications listed here were made in the last 90 days? @lalit97 @Samwalton9 @jsnshrmn your thoughts about this would be appreciated :)

Reply:
Maybe `Recent Applications (last 90 days)`?

Reply:
looks good to me changing the title to this 🙂